src/styles/blocks/main-page.scss

```diff
@@ -0,0 +1,15 @@
+.main-card {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.